benchmark/benchmark.dart

```diff
@@ -32,14 +32,15 @@ final platformPaths = {
 };
 
 /// The command line arguments passed to this script.
-List<String>? arguments;
+late final List<String> arguments;
 
 void main(List<String> args) {
   arguments = args;
 
   for (var style in [p.Style.posix, p.Style.url, p.Style.windows]) {
     final context = p.Context(style: style);
-    final List<String> files = genericPaths.toList()..addAll(platformPaths[style]);
+    final files = genericPaths.toList()
+      ..addAll(platformPaths[style] as List<String>);
```

Comment:
needed the `as` since map[] is always nullable 🤷‍♂

Reply:
what about

```dart
final files = [
  ...genericPaths,
  ...platformPaths[style],
];
```

Reply:
sure – but this is not a code review of ui-as-code features ❗️

Reply:
Does `platformPaths[style]!` not work instead?

Reply:
It didn't when I tried IIRC

Reply:
> Does `platformPaths[style]!` not work instead?

With latest analyzer this works!